README.md

```diff
@@ -4,7 +4,7 @@ An application that shows books and the progress
 
```

Comment:
- [ ] **Optional:** Kindly add more descriptions to your future projects. Your description could capture briefs about other sections of the README. This makes it more of a summary and more professional.